style/constants/color.js

```diff
@@ -12,3 +12,8 @@ export const borderCasing = `hsl(${hueBorderCasing}, 35%, 86%)`;
 export const parkFill = "hsl(136, 41%, 89%)";
 export const parkOutline = "hsl(136, 41%, 79%)";
 export const parkLabel = "hsl(136, 71%, 29%)";
+
+export const airportFill = "hsla(292, 41%, 89%, .3)";
```

Comment:
Alpha blend will be problematic because the color will be different depending on what's behind.  I used the color picker, and the equivalent hsl is `hsl(333, 33%, 93%)`